frontend/src/components/accountPage/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+
+export interface IUserInfoProps {
+  fullName: string;
+  username: string;
+  email: string;
+  studentID: string;
+}
+
+const UserInfo: FC<IUserInfoProps> = props => {
+  const { fullName, username, email, studentID } = props;
+  return (
+    <>
+      <h3>Personal</h3>
+      <p>
+        Full name: <strong>{fullName}</strong>
```

Comment:
I would split into `FirstName` and `LastName` since on the backend it is divided